### PR TITLE
updating caption AR

### PR DESCRIPTION
**Pull request by Adrian-Ryan**
## Pull Request 
This PR updates the caption because it was out of date.
This PR closed / resolves / fixes #11 

## Done
- created a branch
- made a commit

##To 
- [ ] merge the branch
- [ ] delete the ## My Pull Request 

cc @CGreeleyGov

### CODE REVIEW

File: _slides/07-Adrian-Ryan.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+add-caption-Adrian-Ryan
```
Comment (CGreeleyGov) [inline _slides/07-Adrian-Ryan.md:6]: Looks Good

Comment (CGreeleyGov) [inline _slides/07-Adrian-Ryan.md:6]:
```suggestion
Add additional details to your caption
```

**Review by CGreeleyGov — APPROVED** [review]

File: _slides/07-Adrian-Ryan.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (Adomonir) [inline _slides/07-Adrian-Ryan.md:6]: This is good!

File: _slides/07-Adrian-Ryan.md (hunk 1 above)
Comment (Adomonir) [inline _slides/07-Adrian-Ryan.md:6]:
```suggestion
add-caption-Adrian-Ryan, add more details
```

**Review by Adomonir — APPROVED** [review]
LG 😄

**Comment by samuelmhigdon** [comment]
All Good.

**Pull request merged**